oggm/core/preprocessing/climate.py

```diff
@@ -136,6 +136,230 @@ def process_custom_climate_data(gdir):
     out = {'climate_source': fpath, 'hydro_yr_0': y0+1, 'hydro_yr_1': y1}
     gdir.write_pickle(out, 'climate_info')
 
+@entity_task(log, writes=['gcm_data'])
+def process_gcm_data(gdir):
```

Comment:
Currently the function is handling CESM data from the Last Millennial Ensemble. 
I could change the function name into process_cesm_data and give more information in the doc string.